src/components/footer/_macro.njk

```diff
@@ -10,15 +10,15 @@
     {% endif %}
 
     {% set poweredByLogo %}
-   
+
         {% if params.poweredBy is defined and params.poweredBy and params.poweredBy.logo is defined and params.poweredBy.logo %}
             {{
                 onsIcon({
                     "iconType": params.poweredBy.logo,
                     "altText": params.poweredBy.alt | default("Office for National Statistics")
                 })
             }}
-        {% else %}
+        {% elif params.poweredBy is defined and params.poweredBy %}
```

Comment:
This is causing the footers to lose their default ons logo.

Reply:
I think we just need to remove the params for the alt text if on lines 26 and 33, and just hard code them because we know what the icons are.

Reply:
I have managed to fix that but if you're happy hardcoding them then Ill just do that 👍

Reply:
Yeah that's fine